libs/Helpers.py

```diff
@@ -7,4 +8,12 @@ class Helpers:
     def text_to_list(self, text):
         rows = text.split('\n')
         print(rows)
-        return rows
\ No newline at end of file
+        return rows
+
+    @keyword
+    def gte(self, version, target):
```

Comment:
Not too important but how about creating a new file called version_comparer.py (or similar) an name this method `is_gte`?

Reply:
I'm not sure it would provide any benefit?
I think for now we can keep the small helper functions in one file, and eventually break them up later on

Reply:
ok!